### DISCUSSION TITLE
call npm bin script from Deno

### DISCUSSION THREAD

**Discussion by dagda1** [denoland/deno#15706] [Q&A]
I want to use [graphql-code-generator](https://github.com/dotansimha/graphql-code-generator) in deno.

When using it with node, you install it with a package manager and then call a bin script that gets unpacked into `./node_modules/.bin` like this

`yarn graphql-codegen init`

As there is no node_modules in deno, how can I call the executable?

**Comment by bartlomieju** [comment]
`deno run --unstable npm:graphql-codegen init` should work
